POC/ColorPicker/ColorPicker/MainWindow.xaml.cs

```diff
@@ -13,16 +13,21 @@
 using System.Windows.Navigation;
 using System.Windows.Shapes;
 
+using ColorPicker.ColorPickingFunctionality;
+using ColorPicker.ColorPickingFunctionality.SystemEvents;
+
 namespace ColorPicker
 {
     /// <summary>
     /// Interaction logic for MainWindow.xaml
     /// </summary>
     public partial class MainWindow : Window
     {
+        private MouseEvent tempToStopMemoryCleanUp;
         public MainWindow()
         {
             InitializeComponent();
+            tempToStopMemoryCleanUp = new MouseEvent(PixelColorFinder.HandleMouseClick);
```

Comment:
I'm not sure about the name of the variable, seems hacky. A comment would seem better perhaps?

Reply:
I think _mouseEvent would be a good name for example. As @KRMisha said, feel free to explain the purpose of the variable in a comment.

Reply:
Will change